frontend/components/Badge/styles.js

```diff
@@ -1,32 +1,22 @@
 import { css } from 'glamor';
-import { cardBadgeRatio, gridBadgeRatio, pdpBadgeRatio } from '../../config';
+import { badgeWidth, maxBadgeWidth } from '../../config';
 
 const container = css({
-  display: 'inline',
+  display: 'flex',
+  flexDirection: 'column-reverse',
+  alignItems: 'flex-end',
   position: 'absolute',
   right: 0,
   top: 0,
   zIndex: 10,
 }).toString();
 
-const cardImage = css({
-  width: `${cardBadgeRatio.width}`,
-  height: `${cardBadgeRatio.height}`,
-}).toString();
-
-const gridImage = css({
-  width: `${gridBadgeRatio.width}`,
-  height: `${gridBadgeRatio.height}`,
-}).toString();
-
-const pdpImage = css({
-  width: `${pdpBadgeRatio.width}`,
-  height: `${pdpBadgeRatio.height}`,
+const badge = css({
+  width: `${badgeWidth}%`,
+  maxWidth: `${maxBadgeWidth}`,
```

Comment:
I don't think maxBadgeWidth needs to be wrapped in a string. I tried it without and it worked fine. Did you find that it had to be wrapped?

Reply:
In case user inputs 200px instead of 200 the value should be a string